addon/components/o-s-s/banner.hbs

```diff
@@ -1,18 +1,20 @@
-<div class="upf-banner {{if @plain "background-color-gray-50" "background-color-white"}} {{if @selected "upf-banner--selected"}}
-            {{if @disabled "upf-banner--disabled"}} fx-1 fx-row padding-px-18 fx-xalign-center fx-gap-px-12" ...attributes>
+<div class="upf-banner fx-1 fx-row padding-px-18 fx-xalign-center fx-gap-px-12 {{this.modifierClasses}}"
+            ...attributes>
   {{#if (has-block "custom-icon")}}
     <div>{{yield to="custom-icon"}}</div>
   {{else if @icon}}
     <OSS::Icon @style={{fa-icon-style @icon}} @icon={{fa-icon-value @icon}}
-               class="upf-badge upf-badge--shape-round upf-badge--size-md" />
+               class="upf-badge upf-badge--shape-round {{this.iconSizeClass}}" />
   {{else if @image}}
     <img class="upf-badge upf-badge--size-md upf-badge--shape-round" src={{@image}} alt="banner" />
   {{/if}}
-  <div class="fx-col fx-1 fx-gap-px-3">
+  <div class="fx-col fx-1 {{if (not-eq @size 'sm') 'fx-gap-px-3'}}">
     {{#if @title}}
       <span class="font-weight-semibold text-size-5 font-color-gray-900">{{@title}}</span>
     {{/if}}
-    {{#if @subtitle}}
+    {{#if (has-block "secondary-actions")}}
+      <div>{{yield to="secondary-actions"}}</div>
+    {{else if @subtitle}}
```

Comment:
You can't have `secondary-actions` block and subtitle together?

Reply:
No, you can't

Reply:
I guess you can, now ! :trollface: